crates/net/network/src/transactions.rs

```diff
@@ -193,7 +193,8 @@ where
             // we sent a response at which point we assume that the peer is aware of the transaction
             peer.transactions.extend(transactions.iter().map(|tx| tx.hash()));
 
-            let resp = PooledTransactions(transactions);
+            // TODO: remove this! this will be different when we introduce the blobpool
+            let resp = PooledTransactions(transactions.into_iter().map(Into::into).collect());
```

Comment:
how it should work is that the pool should have a function that returns a Vec<PooledTransactionResponse>

so it would be useful to have that in primitives